starters/serverless-apollo-contentful/README.md

```diff
@@ -167,6 +167,26 @@ How to run migrations:
 
 To pre-populate the data within contentful we implement seeding scripts which are located in `scripts/seed`.
 
+### Queueing
+
+Amazon Simple Queue Service (SQS) is a fully managed message queuing service that enables the decoupling of distributed applications and microservices and ensures reliable and scalable processing of application messages.
+
+In this implementation, an SQS message is sent by the handler `sqs-generate-job` via the `sendMessage` function, which is then stored in a queue called `DemoJobQueue`. The SQS handler `sqs-handler` polls this queue and processes any message received.
```

Comment:
> ...is sent by the handler `sqs-generate-job`...

This seems wrong. Not sure what to call `sqs-generate-job`. Could we go with "controller", or maybe "service"?